### ISSUE TITLE
Unused Variable warning

### ISSUE THREAD

**Issue by Sheeri** [WildAid/o-fish-web#277]
**Describe the bug**
When I build the app locally with 'npm start', I get a warning.

**To Reproduce**
1. check out code
2. copy src/config.js.tmpl to src/config.js
3. set realm-app-id in src/config.js to: wildaidsandbox-mxgfy (no other changes needed)
4. npm install && npm start
5. See warning:
**./src/components/charts/charts.component.js
  Line 61:22:  't' is assigned a value but never used**

**Expected behavior**
The warning should not appear.

**Comment by ShalomSagi** [comment]
Can I take this one? 😄

**Comment by Sheeri** [comment]
Hello @ShalomSagi, thanks for offering to work on this! I have assigned it to you.

**Comment by ShalomSagi** [comment]
Hello!
I've followed the provided steps and tried many approaches but I couldn't reproduce the warning.
Everything seems to run perfectly fine on my end.

<img width="518" alt="Screen Shot 2020-10-10 at 15 37 08" src="https://user-images.githubusercontent.com/46726774/95655244-91e3b880-0b0e-11eb-88d0-de0704c8b9cc.png">

**Comment by Sheeri** [comment]
Interesting! Since I opened it yesterday, it seems to have been fixed. I cannot reproduce it either. (I did get another warning, though:
<img width="1091" alt="Screen Shot 2020-10-10 at 9 44 41 AM" src="https://user-images.githubusercontent.com/1767528/95656562-3947f780-0add-11eb-9682-0accc30d977a.png">

I made that into an issue yesterday, but I'm surprised to see you're not getting that.

Anyway, will close this ticket, let us know if you want to work on another.
